com.unity.cinemachine/Runtime/Behaviours/CinemachineFreeLook.cs

```diff
@@ -17,7 +17,9 @@ namespace Cinemachine
     [DisallowMultipleComponent]
     [ExecuteAlways]
     [ExcludeFromPreset]
-    [Obsolete("This is deprecated. Use CmFreelook instead.")]
+    [Obsolete("This is deprecated. Use Create -> Cinemachine -> FreeLook camera, or create a virtual camera with " +
```

Comment:
This is a bit TMI.  "This is deprecated. Use Create -> Cinemachine -> FreeLook camera, or create a virtual camera with CinemachineOrbitalFollow body component" would be enough, I think.